contrib/systemd/conf.py

```diff
@@ -0,0 +1,19 @@
+import shutil
+import intelmq
+
+INTELMQ_DIR = '/opt/intelmq'
+#RUNTIME_CONF = INTELMQ_DIR+'/etc/runtime.conf'
+#PIPELINE_CONF = INTELMQ_DIR+'/etc/pipeline.conf'
+RUNTIME_CONF = intelmq.RUNTIME_CONF_FILE
+PIPELINE_CONF = intelmq.PIPELINE_CONF_FILE
+SYSTEMD_OUTPUT_DIR = INTELMQ_DIR+'/etc/systemd'
+SERVICE_PREFIX = "intelmq."
+DISABLE_IN_CONF = True
+SET_RUNMODE_IN_CONF = True
+INTELMQCTL_BIN = shutil.which('intelmqctl')
+SYSTEMCTL_BIN = shutil.which('systemctl')
```

Comment:
SYSTEMCTL_BIN is the only one which is never used, however it was used for making other units trigger after the collector.